README.md

```diff
@@ -109,7 +109,9 @@ for snaps:
 
 ```
 snap set sctl sctlkey=<YOUR_KMS_KEY_URI>
-snap connect sctl:gcloud
+gcloud auth application-default login
+mkdir -p $HOME/snaps/sctl/current/.config/gcloud
+cp $HOME/.config/gcloud/application_default_credentials.json $HOME/snaps/current/.config/gcloud/application_default_credentials.json
```

Comment:
You should consider using common instead of current. Also it's `snap` singular. Finally, this doesn't instruct the user to `set` the credential config

Reply:
In fact, this help text differs quite a bit from the help text in the command (which is much more concise)

Reply:
ah good catch. I'll get that updated.